godot_project/editor/input_handlers/molecular_structures/add_posing_atom_input_handler.gd

```diff
@@ -86,6 +86,7 @@ func forward_input(in_input_event: InputEvent, _in_camera: Camera3D, out_context
 		_ensure_create_mode()
 	
 	update_preview_position()
+	_update_candidates_if_needed()
```

Comment:
What's the intention of updating candidates when candidates have not changed? you should not need to update the message unless selection have changed again, this looks like unnecesary workload to me 🤔